### PR TITLE
Websockets setup established in Redux + First test reducer action established

### PR DESCRIPTION
**Pull request by dpad-warrior**

### CODE REVIEW

**Review by jackiel12 — APPROVED** [review]
good work!

**Review by mesherrera — APPROVED** [review]

File: src/client/containers/RoomContainer.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```
Comment (wmagee03) [inline src/client/containers/RoomContainer.jsx:1]: What's useEffect?

File: src/client/reducers/playerReducer.js [hunk 2]
```diff
@@ -14,75 +15,118 @@ const playerReducer = (state=initialState, action) => {
 
   switch(action.type) {
     case types.ADD_PLAYER:
-      allPlayers = {...(state.allPlayers)};
```
Comment (wmagee03) [inline src/client/reducers/playerReducer.js:17]: I assume all of this is being put back in right?

**Review by wmagee03 — APPROVED** [review]
Everything looks good from what I can tell, but can I comment back in the playerReducer ?

File: src/client/containers/RoomContainer.jsx (hunk 1 above)
Comment (wmagee03) [reply src/client/containers/RoomContainer.jsx:1]: Nevermind, resolved

**Pull request merged**